js-intro.js

```diff
@@ -68,14 +119,31 @@ var testString = "thisisateststring"
 
 // 4a. Write a function that takes the variable as an argument and returns the string without vowels. Expected output: "thsstststrng"
 
+var testString = "thisisateststring"
 
+var noVowels = function(string) {
+  var vowels = ['a','e','i','o', 'u'];
+  var newStr = "";
+  for (var i=0; i < string.length; i++) {
+    if (!vowels.includes(string[i])) {
+      newStr += string[i]
+    }
+  }
+  return newStr
+}
+console.log(noVowels(testString))
 
+OR
 
-
-// 4b. STRETCH: Update your function to return an error message if the variable is not a string. Copy and paste your function to add these changes.
+noVowels = (str) => {
```

Comment:
This is the or for 4A. & I did figure it out....

var testString = 2

var noVowels = function(string) {
  if (typeof string !== "string") {
  console.log('error, not a string');
  return
}
  var vowels = ['a','e','i','o', 'u'];
  var newStr = "";
  for (var i=0; i < string.length; i++) {
    if (!vowels.includes(string[i])) {
      newStr += string[i]
    }
  }
  return newStr
}
console.log(noVowels(testString))